Add Deployment sorting information to Lineage docs (#2053)
astronomer/docs: astro/data-lineage.md

- When the dataset was accessed.

Click on the name of a dataset to show its lineage graph.

### Filter datasets by Deployment

> To view datasets that were read or written to by a specific Deployment:

Reviewer: ```suggestion
To view datasets that a specific Deployment read or wrote to:
```

1. In the Cloud UI, select a Workspace, click **Deployments**, and then select a Deployment.

> 2. Click **Lineage**. Your Organization **Datasets** tab opens and filters to show only datasets that were read or written to by the selected Deployment.

Reviewer: ```suggestion
2. Click **Lineage**. Your Organization **Datasets** tab opens and filters to show only datasets that the selected Deployment read or wrote to.
```
